package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "postcss-sorting",
   "main": "./lib/postcss-sorting",
-  "version": "2.1.4",
```

Comment:
Let's leave this 'as is'. Because package version increment automatically during `apm publish` process.